SayTheirNames/Source/Controller/Donations/DonationsMoreDetails/DMDCollectionViewDataSource.swift

```diff
@@ -102,8 +103,13 @@ extension DonationsMoreDetailsController: UICollectionViewDataSource {
                 withReuseIdentifier: DMDPhotoSupplementaryView.reuseIdentifier,
                 for: indexPath) as? DMDPhotoSupplementaryView {
                 
-                let person = donation.person
-                person.flatMap { photoView.configure($0) }
+                switch donation.type?.id {
+                case DonationTypes.movement.id:
+                    photoView.configure(withURLString: donation.bannerImagePath)
+                default:
+                    photoView.configure(withURLString: donation.person?.images.first?.personURL)
+                }
```

Comment:
What happens if this image is nil, do we show an empty view? is there a placeholder?

Reply:
Yes, there's already a placeholder image to the UIImageView.